<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyRvDbg</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStep">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDisasmAddr">
        <property name="text">
         <string>80000000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisasm">
        <property name="text">
         <string>Disasm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="tbDisasmResult">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTarget">
    <property name="title">
     <string>Target</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuTarget"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockFunctions">
   <property name="windowTitle">
    <string>Functions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockRegs">
   <property name="windowTitle">
    <string>Regs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableRegs">
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockConsole">
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTextBrowser" name="tbConsole">
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtConsoleCmd"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Memory</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string>80000000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMemRead">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="tbMemory">
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
